lib/brook/driver/kafka/handler.ex

```diff
@@ -3,15 +3,19 @@ defmodule Brook.Driver.Kafka.Handler do
 
   def handle_messages(messages) do
     messages
-    |> Enum.each(fn message -> Brook.process(event(message)) end)
+    |> Enum.each(fn message -> Brook.Event.process(event(message)) end)
 
     :ack
   end
 
-  def event(%{key: type, value: data} = message) do
+  def event(%{key: type, value: value, timestamp: ts} = message) do
+    decoded_json = Jason.decode!(value)
```

Comment:
are you sure you want to use the ! decode for potentially unsanitized input from the library adopters' user inputs?

Reply:
very good point, I am going to leave this for now and come back and look at how to handle errors in a different PR